.gitignore

```diff
@@ -0,0 +1,2 @@
+/*build*
```

Comment:
Looks cmake specific?
We shoud (rather) have "qarma", "qarma-askpass", "zenity", "moc_*", "*.o" and maybe "*tar*" "pkg" and "src"?

Reply:
you can ignore all the worlds you want, a sane build usually is done in a directory.

Reply:
That's besides the point - if we've a gitignore, it should provide all desirable ignores.